frontend/src/main/web/lib/stores/UserMatrixStore.js

```diff
@@ -25,13 +25,17 @@ var _state = {
   }
 };
 
+function statsAPIUrl() {
+  return Configs.baseUrl + "/stats/user/" + Configs.data.profileUser.username + Configs.urlPostfix + "/";
+}
```

Comment:
This function could go below `loadFromServer()` since it is just a helper for that.